website/docs/modules/sources.html.markdown

```diff
@@ -207,6 +207,12 @@ automatically. This is the most common way to access non-public Git
 repositories from automated systems beacuse it is easy to configure
 and allows access to private repositories without interactive prompts.
 
+~> **Note:** The SSH url syntax is different from the SCP syntax (the SCP 
+syntax is what you may be familiar with from GitHub's SSH clone information).
```

Comment:
I think it could read better to put this info together into a single sentence:

> The SSH URL syntax is different from the SCP-style syntax that is given, for example, in GitHub's SSH cloning instructions.